app/connector/sql/src/main/java/io/syndesis/connector/sql/SqlSupport.java

```diff
@@ -49,15 +47,13 @@ public static Connection createConnection(final Map<String, Object> properties)
     }
 
     public static StoredProcedureMetadata getFunctionOrProcedureMetadata(final Connection connection, final String catalog,
```

Comment:
As there is no fetchFunctionColumns call, renaming the method back to getProcedureMetadata and adding a comment about why not using fetchFunctionColumns.